src/common/services/translator.ts

```diff
@@ -15,20 +15,11 @@ import * as frCatalog from "readium-desktop/resources/locales/fr.json";
 import * as deLang from "readium-desktop/resources/locale-names/enLang.json";
 import * as enLang from "readium-desktop/resources/locale-names/enLang.json";
 import * as frLang from "readium-desktop/resources/locale-names/frLang.json";
+import { TFunction } from "readium-desktop/typings/en.translation";
 
 const initI18n = i18n.init({
+    resources: Object.assign(enCatalog, deCatalog, frCatalog),
```

Comment:
I have a request: by design, the various JSON locale files are currently (i.e. `develop` branch) comparable in their canonical form, for example key sorting + value removal/ignoring. This makes it trivial order to detect missing or redundant keys across multiple languages, using off the shelf tooling (as implemented in the `develop` branch). So, although moving the `LANG_CODE.translation` structure into each locale JSON is elegant, this also breaks existing tooling. Is it possible to restore this feature? Or is this change essential to the `I18nTyped` implementation?

Reply:
this change is essential to the I18nTyped implementation, sorry. It's the best practice https://www.i18next.com/principles/translation-resolution. https://github.com/kogai/typed_i18n/issues/66#issuecomment-456303425

Reply:
Ok, we will work around this breaking change, no problems.